docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/Hash.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    // This example demonstrates hashing of categorical string and integer data types.
+    public static class Hash
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext(seed: 1);
+
+            // Get a small dataset as an IEnumerable.
+            var rawData = new[] {
+                new DataPoint() { Category = "MLB" , Age = 18 },
```

Comment:
>Age [](start = 53, length = 3)

I don't think it makes sense to apply hash to a numeric feature, because the value can be used as is and be more useful. It would makes sense to hash an integer column, when the integer is in fact categorical, like "CategoryId", "StoreId" things like that. So I suggest changing Age to CategoryId or CategoryCode.

Reply:
It depends. 

For linear models, usually age is treated as the categorical feature because this way linear model can capture the non-monotonic relationship between the feature and label. Hashing is actually not needed for Age feature as one-hot encoding is sufficient because of low cardinality. 

For trees, Age can be used as-is.

I am not sure if this sample on small data will convey the importance of using hashing. Shall we say something about where hashing is will be useful?

---
In reply to: [268000748](https://github.com/dotnet/machinelearning/pull/3042#discussion_r268000748) [](ancestors = 268000748)

Reply:
Let's do that on the transform documentation, and not the example. 

---
In reply to: [268004737](https://github.com/dotnet/machinelearning/pull/3042#discussion_r268004737) [](ancestors = 268004737,268000748)